server/pulp/server/constants.py

```diff
@@ -2,3 +2,5 @@
 
 PULP_USER_METADATA_FIELDNAME = 'pulp_user_metadata'
 PULP_DJANGO_SETTINGS_MODULE = 'pulp.server.webservices.settings'
+
+PULP_STREAM_REQUEST_HEADER = 'Pulp-Stream-Request'
```

Comment:
I did a bit of research about this because I knew at one point non-standard headers were supposed to start with 'X-'. According to http://tools.ietf.org/html/rfc6648#section-3 new headers should not use the 'X-'.